### ISSUE TITLE
Crashed when test in device

### ISSUE THREAD

**Issue by wenzhaot** [zongren/UmbrellaFramework#1]
dyld: Library not loaded: @rpath/LibrarySample.framework/LibrarySample
  Referenced from: /private/var/containers/Bundle/Application/0BCD0011-8D36-4BA1-AA54-77657F8477DD/ProjectSample.app/Frameworks/BaseSample.framework/BaseSample
  Reason: no suitable image found.  Did find:
	/private/var/containers/Bundle/Application/0BCD0011-8D36-4BA1-AA54-77657F8477DD/ProjectSample.app/Frameworks/BaseSample.framework/Frameworks/LibrarySample.framework/LibrarySample: required code signature missing for '/private/var/containers/Bundle/Application/0BCD0011-8D36-4BA1-AA54-77657F8477DD/ProjectSample.app/Frameworks/BaseSample.framework/Frameworks/LibrarySample.framework/LibrarySample'

**Comment by zongren** [comment]
The project works fine on my mac
![capture](https://user-images.githubusercontent.com/4525819/26868261-f250668c-4b9a-11e7-8ae4-1b8b631c70ef.png)

**Comment by zongren** [comment]
Do feel free to bring up more details 😅

**Comment by wenzhaot** [comment]
Yes, the project works fine on simulator，but crashed on device（iPhone 6p iOS 10.3.2）

**Comment by zongren** [comment]
@wenzhaot 
I will test it later
